libs/sdk-ui-dashboard/src/model/commandHandlers/widgets/validation/insightDrillDefinitionUtils.ts

```diff
@@ -131,7 +132,7 @@ export interface InsightDrillDefinitionValidationData {
     availableDrillTargets: IAvailableDrillTargets;
 }
 
-export const hyperlinkDisplayFormType = "GDC.link";
+export const hyperlinkDisplayFormType = AttributeDisplayFormType.HYPERLINK;
```

Comment:
This assignment seems redundant, can we use `AttributeDisplayFormType.HYPERLINK` directly?